builder/partials/header.hbs

```diff
@@ -14,7 +14,17 @@
         </a>
     </div>
 </header>
-
+<nav class="mdl-card mdl-shadow--2dp card">
+    <select onchange="window.location.href = event.target.value">
```

Comment:
I would prefer to have a normal menu with 3 items because it goes to a new page not filter on the page.

Reply:
done